src/Panel.vue

```diff
@@ -340,10 +340,12 @@
             this.insertCaretInsideHeader(this.$refs.headerWrapper.innerHTML);
         }
       });
-      const panelHeader = this.$slots.header ? this.$refs.headerWrapper.innerHTML : this.headerContent;
-      const panelHeaderText = jQuery(panelHeader).wrap('<div></div>').parent().find(':header').text();
-      if (panelHeaderText) {
+
+      if (this.headerContent) {
+        const panelHeaderText = jQuery(this.headerContent).wrap('<div></div>').parent().find(':header').text();
         this.$refs.cardContainer.setAttribute('id', string(panelHeaderText).slugify().toString());
```

Comment:
you will still need to check `if (panelHeaderText) {`, otherwise it is possible for the `id` to be blank.